oss-vienna-happy-hour.mdx

```diff
@@ -0,0 +1,390 @@
+Unfortunately, I will not be able to see everything, but I made a rough plan.
+Let me know if you are giving a talk and want me to cheer for you! I'm always happy to support a fellow speaker!
+
+Besides the talk sessions I plan to attend, I will also visit the booths and talk to the community.
+One booth I'm particularly excited to visit is the Cloud Native Computing Foundation (CNCF) booth. They were kind enough to send an introductory email, and I plan to visit them in person.
+
+### Sunday 15th
+
+I am arriving on the 15th so first, I will go pick up my badge and you can also do so from 14:00.
+Then I will try to attend Meet Project Margo: Networking Happy Hour. It is open to all OSS attendees.
+
+### Monday 16th
+
+There is a Continental Breakfast between 07:30 and 09:00, I will try to make it there.
+Before breakfast there is a bus tour but it is for brave souls who do not mind waking up before 06:00 to make it to the tour on 06:45.
+
+If you did not pick up on Sunday, you can register and pick up your badge between 07:30 - 17:00.
+
+Then opening ceremony and keynotes start. I like the topic of Keynote: Securing the Software Commons: Standards, Automation, and AI for a Resilient Open Source Future from Abhishek Arya.
+At the same time there is a keynote from Linus Torvalds, so I expect that the attendance of that one will be huge.
+
+I then want to catch the session at 11:20 from Arun Gupta, VP and General Manager for Open Ecosystem at Intel, about Introducing Open Platform for Enterprise AI. I am personally curious about developments in the AI space and would be interested in hearing about the Open Platform for Enterprise AI (OPEA) project.
+There are actually two cool panel sessions colliding with this talk and if you are not so interested in the AI topic, you can search their names in the schedule:
+
+- How to Chart Your Own Career Path in Open Source - Ildiko Vancsa, Open Infrastructure Foundation; Nithya Ruff, Amazon; Allison Randal, Capabilities Limited; and Dawn Foster, CHAOSS
+- What's the State of Open Source in Europe? Key Research Findings Revealed! - Mirko Boehm, Anna Hermansen, Adrienn Lawson & Cailean Osborne, The Linux Foundation
+
+A fun thing I discovered looking at the schedule: There is a project called Basil and the talk is delivered by what I assumed by two Italians.
+So the cutest project name award goes to Basil and if you are curious what is it about search this: "Traceability and Automation Examples With Basil an Open Source Software for Quality Management - Luigi Pellecchia & Gabriele Paoloni, Red Hat"
+
+Then there is a killer slot (12:15) with five talks I could attend, and I know it is not really helpful but here is the list:
+
+<Alert flexDirection="column" alignItems="flex-start">
+  <b>12:15 | Demystifying Secure Application Communication with Zero Trust and Istio Without Sidecars</b>
+
+{' '}
+
+<i>Lin Sun, solo.io</i>
+
+  <span style={{ color: 'var(--chakra-colors-textInfo)' }}>
+  </span>
```

Comment:
@miukimiu do you think we can make the some brand colors instead of this blue? 

Maybe smth like: 
```
<span style={{ color: '#8566FF' }}>
text
</span>
```

Reply:
I was using blue because we use the brand color as a link. So for accessibility, it is better if we don't mix it. Or the text will look like clickable. For example here, we wouldn't know what is clickable.

<img width="826" alt="Screenshot 2024-09-03 at 15 57 39" src="https://github.com/user-attachments/assets/135ee40c-d892-4225-ba96-db5c7bd68303">


So I changed the color to use the `titleAccent` for a11y instead of the blue. It is a very light color but it works side by side with links.

<img width="828" alt="Screenshot 2024-09-03 at 15 56 00" src="https://github.com/user-attachments/assets/0f0694f1-09ce-4465-b403-b5a2582ee4d6">